components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -48,7 +48,7 @@ be lost if SCREAM_HACK_XML is not enabled.
       <sections>ctl_nl</sections>
     </file>
     <file name="data/scream_input.yaml" format="yaml">
-      <sections>driver_options,iop_options,atmosphere_processes,grids_manager,initial_conditions,Scorpio,e3sm_parameters</sections>
+      <sections>driver_debug_options,driver_options,iop_options,atmosphere_processes,grids_manager,initial_conditions,Scorpio,e3sm_parameters</sections>
```

Comment:
@mahf708 I don't think it is trivial.  I tried with just the above and hit an XML error.  I tried a Google search and it looks like it is up to the interpreter if it catches the line breaks.  So we may be stuck with a single long line for now.  Although I agree it would be easier to read if we could break it down into multiple lines.

Reply:
I think the issue may be in how we parse the XML node content to generate the yaml/nml files. We may be able to modify that code, so that `     a,\n     b` is treated the same as `a,b`. But probably not in this PR.